dv/uvm/Makefile

```diff
@@ -103,6 +103,7 @@ iss_sim:
 	cd ${GEN_DIR}; \
 	python3 ./run.py \
      --o=${OUT}/instr_gen ${GEN_OPTS} \
+     --riscv_dv_root=${GEN_DIR} \
```

Comment:
Is this really needed? can you try remove the "cd xxx" command, and just do python3 ${GEN_DIR}/run.py?

Reply:
I will approve this PR to get regression failure fixed, please try the option I mentioned later.

Reply:
Ack, I've opened a new PR to streamline this script invocation.